### PR TITLE
Replace fork broccoli by published version

### LINKED ISSUE #175: Replace the fork of broccoli-svgstore with a published plugin

### PR DESCRIPTION
**Pull request by dafortin**
#fix#

Fix #175

# CHANGELOG
* **Update** `broccoli-svgstore` to a published version of the package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,6 +103,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
+    "broccoli-svgstore": "https://github.com/sglanzer/broccoli-svgstore/tarball/master",
```
Comment (rox163) [inline package.json]: confused here...still using the fork? looks like it was added here?

Comment (dafortin) [reply package.json]: weird :O

Comment (dafortin) [reply package.json]: Fixed

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/288/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Pull request merged**